lms/djangoapps/instructor_task/tests/test_tasks.py

```diff
@@ -222,6 +222,23 @@ def _test_run_with_short_error_msg(self, task_class):
 class TestRescoreInstructorTask(TestInstructorTasks):
     """Tests problem-rescoring instructor task."""
 
+    def assert_task_output(self, output, **expected_output):
+        """
+        Check & compare output of the task
+        """
+        self.assertEquals(output.get('total'), expected_output.get('total'))
+        self.assertEquals(output.get('attempted'), expected_output.get('attempted'))
+        self.assertEquals(output.get('succeeded'), expected_output.get('succeeded'))
+        self.assertEquals(output.get('skipped'), expected_output.get('skipped'))
+        self.assertEquals(output.get('failed'), expected_output.get('failed'))
+        self.assertEquals(output.get('action_name'), expected_output.get('action_name'))
+        self.assertGreater(output.get('duration_ms'), expected_output.get('duration_ms', 0))
+
+    def get_task_output(self, task_id):
+        """Get and load instructor task output"""
+        entry = InstructorTask.objects.get(id=task_id)
```

Comment:
@attiyaIshaque  As this method is in the tests, passing incorrect task_id will eventually fail the test. Any test will not pass until passing a correct `task_id` which indicates to me that we shouldn't he handing the scenario here.
Let me know what you think.